src/FrameReflower/Block.php

```diff
@@ -521,11 +520,10 @@ protected function _text_align()
 
                 // Adjust the last line if necessary
                 if ($last_line->left) {
-                    foreach ($last_line->get_frames() as $frame) {
-                        if ($frame instanceof BlockFrameDecorator) {
-                            continue;
+                    foreach ($line->get_frames() as $frame) {
```

Comment:
Should this be `$last_line->get_frames()`?

Reply:
Good catch, thanks!

Reply:
Fixed